stdlib/public/Concurrency/Task.swift

```diff
@@ -31,20 +41,52 @@ import Swift
 /// These partial periods towards the task's completion are `PartialAsyncTask`.
 /// Partial tasks are generally not interacted with by end-users directly,
 /// unless implementing a scheduler.
-public enum Task {
+public struct Task {
+  internal let _task: Builtin.NativeObject
+
+  // May only be created by the standard library.
+  internal init(_ task: Builtin.NativeObject) {
+    self._task = task
+  }
+}
+
+// ==== Current Task -----------------------------------------------------------
+
+extension Task {
+
+  /// Returns 'current' `Task` instance, representing the task from within which
+  /// this function was called.
+  ///
+  /// All functions available on the Task
+  // TODO: once Kavon's async properties land make this computed property
+  public static func current() async -> Task {
```

Comment:
Please use `Builtin.getCurrentAsyncTask()`, because `unsafeCurrent` is not yet implemented.

Reply:
Done, not implemented APIs now have `  @available(*, deprecated, message: "Not implemented yet, use Builtin.getCurrentAsyncTask() or Task.___unsafeCurrentAsync() until this function is implemented.")` and we're using the Task.___unsafeCurrentAsync() in tests etc for now.

The changes: https://github.com/apple/swift/pull/35704/commits/9e18cbd217fcfb6d66b646f9ca6212bdd0c588ca